ui/tract-renderer/src/main/java/org/cru/godtools/tract/analytics/model/ContentAnalyticsActionEvent.kt

```diff
@@ -14,8 +14,8 @@ class ContentAnalyticsActionEvent(@VisibleForTesting internal val event: Analyti
     override val adobeAttributes: Map<String, *>? get() = event.attributes
     override val firebaseEventName = if (isForSystem(AnalyticsSystem.ADOBE)) {
         Timber.tag(TAG).e(UnsupportedOperationException("XML Adobe Analytics Usage"), "action ${event.action}")
-        event.action?.replace(Regex("[ \\-]"), "_").orEmpty()
+        event.action?.replace(Regex("[ \\-]"), "_").orEmpty().toLowerCase()
```

Comment:
Clark requested that all the events are lowercased for consistency so I went ahead and did that here.

Reply:
alright, this is obviously a temporary solution, we will need to start renaming the events when we perform the analytics cleanup phase